README.md

```diff
@@ -1,54 +1,149 @@
+
+### GNU+Linux and macOS
+
+*Compiling* requires MSBuild and *running* requires Mono and WINE, but **BizHawk does not run under WINE** — only the bundled libraries are required.
+
+Building is as easy as:
+```sh
+git clone https://github.com/TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+msbuild --release BizHawk.sln
+```
+
+Remove the `--release` flag from MSBuild if you want debugging symbols.
+
+TODO it's not --release
+
+If your distro isn't listed above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see the *Installing* section above, substituting the repo's `output` folder for the download. If your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+### Windows 7/8.1/10
+
+TODO
+
+[Compiling](http://tasvideos.org/Bizhawk/Compiling.html)
+
+## Usage (casual)
+
+TODO
+
+## Usage (advanced)
+
+TODO
+
+[Commandline](http://tasvideos.org/Bizhawk/CommandLine.html)
+
+[CompactDiscInfoDump](http://tasvideos.org/Bizhawk/CompactDiscInfoDump.html)
+
+[Rerecording](http://tasvideos.org/Bizhawk/Rerecording.html)
+
+[TAS movie file format](http://tasvideos.org/Bizhawk/TASFormat.html)
+
+## Support and troubleshooting
+
+A short [FAQ](http://tasvideos.org/Bizhawk/FAQ.html) is provided on the [BizHawk wiki](http://tasvideos.org/Bizhawk.html). If your problem is one of the many not answered there, and you can't find it in the [issue tracker search](https://github.com/TASVideos/BizHawk/issues?q=is%3Aissue+ISSUE_KEYWORDS), check the [BizHawk forum](http://tasvideos.org/forum/viewforum.php?f=64) at tasvideos.org, or ask on IRC:
+
+[![IRC | chat.freenode.net #bizhawk](https://img.shields.io/badge/IRC-chat.freenode.net_%23bizhawk-steelblue.svg?style=flat)](ircs://chat.freenode.net:6697/bizhawk)
+[![Matrix (IRC bridge) | #freenode_#bizhawk:matrix.org](https://img.shields.io/badge/Matrix_(IRC_bridge)-%23freenode__%23bizhawk:matrix.org-mediumpurple.svg?logo=matrix&style=flat)](https://matrix.to/#/#freenode_#bizhawk:matrix.org)
+[![freenode webchat | #bizhawk](https://img.shields.io/badge/freenode_webchat-%23bizhawk-lightcoral.svg?style=flat)](http://webchat.freenode.net/?channels=bizhawk)
+
+## Contributing
+
+BizHawk is Open Source Software, so you're free to modify it however you please, and if you do, we invite you to share! Under the MIT license, this is *optional*, just be careful with reusing cores as some have copyleft licenses.
```

Comment:
Might want to add that when one is committing to the main project directly, we do not merge, we fetch and rebase. And even then, bigger changes should be PRs.

Reply:
not sure that's relevant because you're not giving out push access?

Reply:
It's for those who have it already. Not sure where to put it.

Reply:
don't you think people you're giving push access aught to be told directly how the repo is managed? this is for new contributors